main.go

```diff
@@ -138,6 +129,33 @@ func status(response http.ResponseWriter, request *http.Request, config Config)
 	response.Write(result)
 }
 
+func pixieHandler(response http.ResponseWriter, request *http.Request, config Config) {
+
+	macaddr := mux.Vars(request)["macaddr"]
+	hostname, found := config.MachineBuild[macaddr]
+
+	if found == false {
```

Comment:
Yeah, I tried but a map lookup gives you back true or false, not an error, hence the found.